extra/utils/permissions.js

```diff
@@ -0,0 +1,36 @@
+const permissions = {
+     'getUsers': {
+     all: ['head-trainer','manager'],
+     read : ['trainee', 'trainer'],
+     write : ['trainer'],
+     delete: [],
+     },
+     
+   }
+
+function hasPermissions(moduleName,role,permissionType)
```

Comment:
Hi shikhar, as per the AC, your function need to return the result either true or false... please correct it